fish-rust/src/util.rs

```diff
@@ -12,12 +13,30 @@ mod ffi {
     }
 
     extern "Rust" {
-        fn wcsfilecmp(a: wcharz_t, b: wcharz_t) -> i32;
-        fn wcsfilecmp_glob(a: wcharz_t, b: wcharz_t) -> i32;
+        #[cxx_name = "wcsfilecmp"]
+        fn wcsfilecmp_ffi(a: wcharz_t, b: wcharz_t) -> i32;
```

Comment:
nice, I didn't know about `cxx_name`

Reply:
Took me quite a while to find this in the cxx docs too.